aws/cloudformation/data.yml.erb

```diff
@@ -97,27 +97,17 @@ Resources:
             Resource:
               - 'arn:aws:s3:::cdo-data-sharing-internal/*'
               - 'arn:aws:s3:::cdo-data-sharing-internal'
-  RDSEndpoint:
+  ProductionMySQLDatabaseReportingEndpoint:
     Type: AWS::DMS::Endpoint
     Properties:
-      EndpointIdentifier: !Sub "RDS-${AWS::StackName}"
-      EndpointType: source
-      EngineName: mysql
-      ServerName: !Ref RDSHost
-      Port: 3306
-      Username: !Ref RDSUsername
-      Password: !Ref RDSPassword
-      ExtraConnectionAttributes: ResumeFetchForXRows=0;UnloadTimeout=86400
-  AuroraCloneEndpoint:
-    Type: AWS::DMS::Endpoint
-    Properties:
-      EndpointIdentifier: !Sub "AuroraCloneEndpoint-${AWS::StackName}"
+      EndpointIdentifier: !Sub "MySQLReportingEndpoint-${AWS::StackName}"
       EndpointType: source
       EngineName: aurora
-      # At the time this template is rendered the Redshift Import Clone may not exist, so we can't use the RDS describe
-      # endpoints API to get this server name.
-      ServerName: !Sub "<%= RedshiftImport::CLONE_CLUSTER_ID %>.${RDSClusterEndpointRootDomain}"
+      # The Stack Parameter RDSHost must be set to the domain name of the production RDS Proxy reporting endpoint.
```

Comment:
Move/Copy this to the parameter description.

Reply:
Done!